### DISCUSSION TITLE
How to test the v4 version?

### DISCUSSION THREAD

**Discussion by brunowilliang** [nativewind/nativewind#595] [General]
Hello everyone!

I'm looking forward to version 4 of Nativewind that is in development. Could anyone share details about the changes and new features? Any documentation or preliminary information would be greatly appreciated. Thank you!

**Comment by MPJHorner** [comment]
install on npm from here (update your package.json) https://www.npmjs.com/package/nativewind/v/4.0.1

**Comment by MPJHorner** [comment]
new documentation here https://www.nativewind.dev/v4/overview 

~ haven't been able to find a list of changes however

**Comment by danstepanov** [comment]
Closing this discussion, if this is still relevant, feel free to open a new one.
